SmartcarAuth/SmartcarAuth.swift

```diff
@@ -144,19 +143,32 @@ Smartcar Authentication SDK for iOS written in Swift 3.
         let urlComp = URLComponents(url: url, resolvingAgainstBaseURL: false)
 
         guard let query = urlComp?.queryItems else {
-            return completion(AuthorizationError.missingQueryParameters, nil, nil)
+            return completion(AuthorizationError.missingQueryParameters, nil, nil, nil)
         }
 
         let queryState = query.filter({ $0.name == "state"}).first?.value
-
-        if query.filter({ $0.name == "error"}).first?.value != nil {
-            return completion(AuthorizationError.accessDenied, nil, queryState)
+        var vehicle = Vehicle()
```

Comment:
As discussed offline, this should ideally be VehicleInfo so as to not introduce a new concept unnecessarily

Reply:
was changed back to vehicleInfo